panel/io/location.py

```diff
@@ -58,7 +56,7 @@ class Location(Syncable):
         should be set to False""")
 
     # Mapping from parameter name to bokeh model property name
-    _rename = {"name": None}
+    _rename: Mapping[str, Union[str, None]] = {"name": None}
```

Comment:
I used `Mapping[str, Union[str, None]]` type annotation instead of  `KeysAndGetItem[str, Union[str, None]]` as it was already used it the Panel code. I am not sure if its the right choice though.

Reply:
I presume `_rename` is a static member. Then the type should be wrapped with `ClassVar` to indicate this (everywhere).

Reply:
`KeysAndGetItem` seems to be an oddly specific type (where does it come from btw.?). `Mapping` is fine in this context, but it really depends on the interface you want to expose. The smaller the surface of the API, the better. One can always expand it, by changing the type to the more specific `Dict[..., ...]`, but not the other way, at least without breaking compatibility.

Reply:
`KeysAndGetItem` was the type reported by mypy as the expected type. It was inferred from the attribute value being a dictionary.

Thanks for mentioning ClassVar. I learned something new there.

Reply:
Yep, also just learned about ClassVar.